week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,102 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+console.clear();
+let shakingBallAnswer;
+
+function shakeBall() {
+  let shakingBall = Math.round(Math.random() * 20);
+  switch (shakingBall) {
+    case 1:
+      shakingBallAnswer = "It is certain.";
```

Comment:
How come you decided to create a global variable `shakingBallAnswer` instead of passing in a parameter which you can modify? What do you think is the better practice?